gooey/gui/components/widgets/bases.py

```diff
@@ -183,7 +183,7 @@ def getUiState(self) -> t.FormField:
     def syncUiState(self, state: FormField):  # type: ignore
         self.widget.setValue(state['value'])  # type: ignore
         self.error.SetLabel(state['error'] or '')
-        self.error.Show(state['error'] is not None and state['error'] is not '')
+        self.error.Show(state['error'] is not None and state['error'] != '')

```

Comment:
isn't it enough to write

```
self.error.Show(bool(state['error']))
```
?

Reply:
That seems to work as well, assuming we want other falsy values to equate to false as well (such as 0).